docs/src/ann_indexes.md

```diff
@@ -24,18 +24,27 @@ Lance supports `IVF_PQ` index type by default.
     ```python
     import lancedb
     import numpy as np
+    from lancedb.index import IvfPq
+
     uri = "data/sample-lancedb"
-    db = lancedb.connect(uri)
 
     # Create 10,000 sample vectors
     data = [{"vector": row, "item": f"item {i}"}
         for i, row in enumerate(np.random.random((10_000, 1536)).astype('float32'))]
-
+    
+    # Synchronous client
+    db = lancedb.connect(uri)
     # Add the vectors to a table
     tbl = db.create_table("my_vectors", data=data)
-
     # Create and train the index - you need to have enough data in the table for an effective training step
     tbl.create_index(num_partitions=256, num_sub_vectors=96)
+
+    # Asynchronous client
+    async_db = await lancedb.connect_async(uri)
+    # Add the vectors to a table
+    async_tbl = await async_db.create_table("my_vectors", data=data)
+    # Create and train the index - you need to have enough data in the table for an effective training step
+    await async_tbl.create_index("vector", config=IvfPq(num_partitions=256, num_sub_vectors=96))
```

Comment:
Could we do tabs for async vs sync? I worry having both and using comments will make the docs pages long and cluttered. Easier if someone can choose up front which they want and then all the following examples will keep with that language.